pegasus/sites.v3/code.org/public/global/fa/about.haml

```diff
@@ -0,0 +1,92 @@
+---
+title: Code.org in Farsi
+theme: responsive_full_width
+set_dir: true
+---
```

Comment:
This will make the page title translated, and it can be used across the other pages too w/ their respective strings!

```suggestion
---
theme: responsive_full_width
set_dir: true
---

:ruby
  @header["title"] = hoc_s(:farsi_page_top_heading)
```